src/lib.rs

```diff
@@ -566,9 +563,9 @@ impl<'a> ParsedMail<'a> {
     ///             "\n",
     ///             "This is the body").as_bytes())
     ///         .unwrap();
-    ///     assert_eq!(p.get_body().unwrap(), "This is the body");
+    ///     assert_eq!(p.get_body().unwrap(), b"This is the body");
     /// ```
-    pub fn get_body(&self) -> Result<String, MailParseError> {
+    pub fn get_body(&self) -> Result<Vec<u8>, MailParseError> {
```

Comment:
I would prefer if we left the string-returning version as get_body, and made the Vec<u8>-returning version get_body_raw or get_body_as_bytes or something along those lines. That way existing callers that use this function don't need to get renamed.